### DISCUSSION TITLE
Bug in displaying issues links in edit mode

### DISCUSSION THREAD

**Discussion by PavelMal** [allure-framework/allure2#2082] [TestOps]
Hi there!
Allure TestOps version: 4.20.2
Got an empty string in edit mode of issue links

Steps to reproduce:
1. Create a case and link an issue (for example Jira integration)
2. Click on edit button in issue links

**Actual result:** Got the same number of issues links (in my case 3) but they are empty
**Expected result:** Got the same number of issues links and they are filled with the same links as you can see on case page

<img width="1262" alt="image" src="https://github.com/allure-framework/allure2/assets/12068504/13b64037-a4db-407a-b681-f2da9a7f02d9">

<img width="1662" alt="image" src="https://github.com/allure-framework/allure2/assets/12068504/2d1c1e71-e9aa-4748-82de-3a2e131c7507">

**Comment by cheshi-mantu** [comment]
@PavelMal Hello sir,
please consider using https://help.qameta.io for the bug reports, this place is rather for seeking help and discussion and Allure Testops as paid product has dedicated support portal.
This issue has been already fixed in newer release: https://docs.qameta.io/allure-testops/release-notes/#🐞-bug-fixes
